demos/dotnet/dotnet/ProjectProperties.cs

```diff
@@ -9,6 +9,7 @@ class ProjectProperties
     {
         public string AssemblyName;
         public string OutputDirectory;
+        public string OutputType;
```

Comment:
I used the Common MSBuild Project Properties as an example (https://msdn.microsoft.com/en-us/library/vstudio/bb629394(v=vs.110).aspx) which had OutputType to 'Specifies the file format of the output file.'